tests/torch/test_compile_torch.py

```diff
@@ -1056,7 +1056,7 @@ def test_net_has_no_tlu(
     if num_inputs > 1 and use_qat:
         return
 
-    use_conv = isinstance(input_shape, tuple) and len(input_shape) > 1
+    use_conv = isinstance(input_shape, tuple) and len(input_shape) == 3
```

Comment:
What's happening?

Reply:
I can change that back but I found it confusing when I was debugging

Reply:
a comment could be useful.

alternatively you can put the `use_conv` as a constant in the pytest parameter list just like `use_qat`

Reply:
I was just curious it's weird to see hard coded value change like this. Thanks

Reply:
Changed it back to what it was